src/shared-with-pluot-core/selectors.js

```diff
@@ -57,20 +57,19 @@ const _getIsSubscribedToTrack = (p, p2id, mediaTag) => {
   if (!p) {
     return false;
   }
-  if (
-    !p.public.subscribedTracks || // sanity check
-    p.public.subscribedTracks.ALL
-  ) {
-    return true;
+  const sTracks = p.public.subscribedTracks;
+  // Below shows the return values for all the various versions of sTracks
+  //   { ALL: true }                -> true
+  //   { ALL: false }               -> false
+  //   undefined                    -> true
+  //   {},                          -> false
+  //   { p2id: { mediaTag: true }}  -> true
+  //   { p2id: { mediaTag: false }} -> false
```

Comment:
note: i'm still a little hazy on what a value of 'avatar' would/should mean in this function